synapse/cortex.py

```diff
@@ -5159,12 +5159,27 @@ async def runStormDmon(self, iden, ddef):
         return dmon
 
     async def getStormDmon(self, iden):
+        mirurl = self.conf.get('mirror')
+        if mirurl is not None:
+            async with await s_telepath.openurl(mirurl) as leader:
+                return await leader.getStormDmon(iden)
```

Comment:
Any interest in making this a decorator (`@leader` or something) so it can be easily added to other methods that need to query up to the leader?

Reply:
Yeah I could see that being nice to have